src/maker.rs

```diff
@@ -298,7 +304,7 @@ mod tests {
         bitcoin::Amount::from_btc(btc).unwrap()
     }
 
-    fn dai(dai: f64) -> dai::Amount {
+    fn dai_amount(dai: f64) -> dai::Amount {
```

Comment:
🙃  I'd change this to `dai(...) -> dai::Asset`, the point of this function was that we don't have to care what is behind it.